modules/buttons.js

```diff
@@ -0,0 +1,10 @@
+const buttons = `

```

Comment:
Your modules should be functions. You can watch the course I shared to see how and why.